pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -136,12 +136,27 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [
     "ibis"
-  ] ++ (map (backend: "ibis.backends.${backend}") backends);
+  ] ++ map (backend: "ibis.backends.${backend}") testBackends;
 
   meta = with lib; {
     description = "Productivity-centric Python Big Data Framework";
     homepage = "https://github.com/ibis-project/ibis";
     license = licenses.asl20;
+    passthru = {
```

Comment:
passthru goes onto the same level as meta, not into meta.

Reply:
Fixed, thanks!